src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -58,6 +59,10 @@ public class MainWindow extends UiPart<Stage> {
     @FXML
     private StackPane statusbarPlaceholder;
 
+    @FXML
+    private TabPane tabPane;
+
```

Comment:
This variable is currently unused in this PR, but I am assuming this is to be used for switching panes in the future - along with the change to `MainWindow.fxml`.

Helpful link for future reference: https://stackoverflow.com/questions/6902377/javafx-tabpane-how-to-set-the-selected-tab